src/test/java/org/apache/commons/compress/compressors/snappy/FramedSnappyCompressorInputStreamTest.java

```diff
@@ -184,14 +184,11 @@ public void testUnskippableChunk() {
             (byte) 0xff, 6, 0, 0, 's', 'N', 'a', 'P', 'p', 'Y',
             2, 2, 0, 0, 1, 1
         };
-        try {
-            final FramedSnappyCompressorInputStream in =
-                new FramedSnappyCompressorInputStream(new ByteArrayInputStream(input));
-            in.read();
-            fail("expected an exception");
-            in.close();
+        try (final FramedSnappyCompressorInputStream in =
+                     new FramedSnappyCompressorInputStream(new ByteArrayInputStream(input))) {
+            IOException exception = assertThrows(IOException.class, () -> in.read());
```

Comment:
:+1: you've already replaced that `fail` by an `assertThrows` here. We should probably change it in the other places too.